packages/legend-extension-dsl-serializer/src/models/protocols/pure/v1/transformation/pureGraph/V1_DSLSerializer_GraphBuilderHelper.ts

```diff
@@ -24,13 +24,14 @@ import type {
   PackageableElementImplicitReference,
   V1_GraphBuilderContext,
 } from '@finos/legend-graph';
+import { OptionalPackageableElementExplicitReference } from '@finos/legend-graph';
 
 export const V1_resolveSchemaSet = (
```

Comment:
The thing about this utility function is you'd probably don't need it here if it resolves to an optional reference, because not all the places that need to resolve a `SchemaSet` has it as `OptionalPackageableElementReference<SchemaSet>`. If only one place uses this (I suppose when you build graph, let's kill this method here and just move this creation directly to the builder)